exp/topology/desiredstate/desired_state_test.go

```diff
@@ -380,7 +399,7 @@ func TestComputeControlPlane(t *testing.T) {
 		scope := scope.New(cluster)
 		scope.Blueprint = blueprint
 
-		r := &Reconciler{}
+		r := &generator{}
```

Comment:
nit:
```suggestion
		g := &generator{}
```

Same for others in this file

Reply:
Got rid of the variable. g is already Gomega in a lot of cases